telluric/georaster.py

```diff
@@ -83,6 +84,15 @@ class PixelStrategy(Enum):
     FIRST = 1
 
 
+def join(rasters):
+    """
+    This method takes a list of rasters and a raster that is consturcted of all of them
+    """
+    bounds = [raster.footprint() for raster in rasters]
+    bounds = reduce((lambda x, y: x+y), bounds).cascaded_union
```

Comment:
can you explain why `bounds.cascaded_union` is not enough?

Reply:
bounds is a list and not a feature collection so it doesn't have `cascaded_union` 

I could write this code differently:
```
bounds = tl.FeatureCollection.from_geovectors([raster.footprint() for raster in rasters]).cascaded_union

```

but it is not a big difference

Reply:
I would prefer the second approach, because the `+` might be expensive in a big list and it may add calculation errors.

Reply:
the thing is that I can't import FeatureCollection globally on the file, because of the cyclic dependencies so checkout the code now